src/app/highchart/highchart.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit } from '@angular/core';
+import { Chart } from 'angular-highcharts';
+import { UserService } from '../user.service';
+import { USERS } from '../mock-user';
+import { getTreeNoValidDataSourceError } from '@angular/cdk/tree';
+import { useAnimation } from '@angular/animations';
+
+@Component({
+  selector: 'app-highchart',
+  templateUrl: './highchart.component.html',
+  styleUrls: ['./highchart.component.sass', './highchart.component.css']
+})
+export class HighchartComponent implements OnInit {
+ // series: any[];
+  arr =  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+  arr1 =  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+ users = USERS;
+  chart = new Chart( {
+    chart: {
+      type: 'column'
+    },
+    title: {
+      text: 'Annual Expenses month wise'
+    },
+    credits: {
+      enabled: false
+    },
+    xAxis: {
+      categories: [
+          'Jan',
+          'Feb',
+          'Mar',
+          'Apr',
+          'May',
+          'Jun',
+          'Jul',
+          'Aug',
+          'Sep',
+          'Oct',
+          'Nov',
+          'Dec'
+      ],
+      crosshair: true
+    },
+    yAxis: {
+        min: 0,
+        title: {
+            text: 'Monthly Expenses (INR in thousands)'
+        }
+    },
+  plotOptions: {
+      column: {
+          pointPadding: 0.2,
+          borderWidth: 0
+      }
+  },
+    series: [ {
+        name : 'complete',
+        data : this.getdata(),
+      },
+      {
+        name : 'uncomplete',
+        data :  this.getdata1(),
```

Comment:
use function call after the constructor, not at variable declaration

Reply:
done